pyproject.toml

```
@@ -76,6 +76,17 @@ pyrs = { path = ".", editable = true }
 hatchling = "*"
 versioningit = "*"
 
+[tool.pixi.package.run-dependencies]
+mantidworkbench = ">=6.13"
```

```suggestion
mantidworkbench =">=6.13.0,<6.14.0"
```
This will avoid inadvertently deploying mantid 6.14 to production